stock_picking_reorder_lines/README.rst

```diff
@@ -11,14 +13,32 @@ For further information, please visit:
 
 * https://www.odoo.com/forum/help-1
 
+Installation
+============
+
+To install this module, you need to install its dependant module:
+
+ * stock_picking_reorder_lines (https://github.com/OCA/stock-logistics-workflow/)
```

Comment:
This module is stock_picking_reorder_line, so there's no real dependency. Remove this.